packages/core/babel.config.js

```diff
@@ -1,3 +1,9 @@
 module.exports = {
   presets: ['module:metro-react-native-babel-preset'],
+  plugins: [
+    ['@babel/plugin-transform-class-properties', { loose: true }],
+    ['@babel/plugin-transform-private-methods', { loose: true }],
+    ['@babel/plugin-transform-private-property-in-object', { loose: true }]
+  ],
```

Comment:
Why are these plugins needed?

Reply:
I added those to fix test failures in `test/tracing/reactnavigation.ttid.test.tsx`
```
Class private methods are not enabled. Please add `@babel/plugin-transform-private-methods` to your configuration.
```